### DISCUSSION TITLE
IRHVAC with MQTT create wrong Items

### DISCUSSION THREAD

**Discussion by heitec11** [arendst/Tasmota#19047] [Q&A]
Tasmota 13.0 IR
Send a command by IRHVAC creates 3 MQTT items with are not for AC:

ESP-TCL-IR.IRHVAC_Light
ESP-TCL-IR.IRHVAC_Mode
ESP-TCL-IR.IRHVAC_Power

with no values.

Send command was:
IRhvac {"Vendor":"TCL112AC","Model":"TAC09CHSD", "Power":"On","Mode":"Cool","FanSpeed":"Auto","Temp":22}

**Comment by sfromis** [comment]
What do you mean by "creates 3 MQTT items"? Where do you see those, in what format? As a starting point, look for what Tasmota actually publishes, visible in the Tasmota console.
